2d-survival-template/project/player.gd

```diff
@@ -26,6 +26,15 @@ func _physics_process(delta: float) -> void:
   velocity = speed * input
   move_and_slide()
 
+  if abs(velocity) > Vector2.ZERO and !is_moving:
+    is_moving = true
+    GlobalSignalBus.move_started.emit(self)
+  
+  if velocity == Vector2.ZERO:
```

Comment:
@loteque 
It will keep getting called each frame.
I think we need to add the condition `not is_moving`.

Reply:
an early return?

Reply:
done. https://github.com/loteque/2024_oh_jam/pull/1/commits/b1e2b73af5fb083c178b83c9c4888e77f350265a